app/serializers/current_user_serializer.rb

```diff
@@ -338,4 +340,12 @@ def redesigned_user_menu_enabled
   def likes_notifications_disabled
     object.user_option&.likes_notifications_disabled?
   end
+
+  def include_all_unread_notifications?
```

Comment:
Should we exclude the fields that are not used when the new user menu is enabled too?

Reply:
Same as https://github.com/discourse/discourse/pull/17718#discussion_r935190653; I'm worried about themes/plugins that expect the old fields breaking when we enable the new menu. I'd like to leave them alone for now.

Reply:
Ahh ok gotcha :+1: